gitrise.sh

```diff
@@ -167,9 +200,16 @@ function generate_build_payload() {
 {
   "build_params": {
     "branch": "$BRANCH",
+    "branch_dest": "$BRANCH_DEST",
     "commit_hash": "$COMMIT",
+    "commit_message": "$MESSAGE",
```

Comment:
I've tested this implementation and this will not work when your commit message contains spaces. I changed the `generate_build_payload()` implementation so it used jq to create the payload.

```
function generate_build_payload() {
    local environments=$(process_env_vars "$ENV_STRING")
    jq -n --arg branch "$BRANCH" \
          --arg commit "$COMMIT" \
          --arg message "$MESSAGE" \
          --arg tag "$TAG" \
          --arg workflow "$WORKFLOW" \
          --argjson environments "$environments" \
    '{
        build_params: {
            branch: $branch,
            commit_hash: $commit,
            commit_message: $message,
            tag: $tag,
            workflow_id: $workflow,
            environments: $environments
        },
        hook_info: {
            type: "bitrise"
        }
    }'
}
```